src/withRoundTrip.ts

```diff
@@ -18,19 +19,24 @@ function createChange(options: ChangeOptions) {
   return { ...options, time };
 }
 
+// https://developer.mozilla.org/en-US/docs/Web/API/Node/nodeType#node_type_constants
+function isElement(node: Node): node is Element {
+  return node && node.nodeType === 1;
+}
+
 export const withRoundTrip = (storyFn: StoryFunction) => {
   const emit = useChannel({});
 
   React.useEffect(() => {
     const observer = new MutationObserver((mutationsList) => {
       for (const mutation of mutationsList) {
-        // @ts-ignore
-        // use TS appropriate way to gather attribute
-        let assertiveness = mutation.target?.getAttribute("aria-live");
+        let assertiveness = isElement(mutation.target)
```

Comment:
it's a bummer that there isn't a super tidy way to do this but i think i understand the utility here. we need to capture "assertiveness" in a way that can fail reliably.

Reply:
Yup - you need to narrow the type of `Node` here in order to have access to `getAttribute`.